mlir/include/mlir/Dialect/Linalg/IR/LinalgRelayoutOps.td

```diff
@@ -190,7 +190,30 @@ def Linalg_PackOp : Linalg_RelayoutOp<"pack", [
     // Method to get the `RankedTensorType` of the result based on the inner
     // tiles, position of the inner tiles (innerDimsPos)  and interchange vector
     // of outer loops (outerDimsPerm).
-    static RankedTensorType inferPackedType(RankedTensorType sourceType,
+    /// This method uses inferPackedShape to ensure consistency with other shape
+    /// inference methods regarding which dimensions are dynamic.
```

Comment:
This is implementation details which should not be documented here. I think we can just remove it..